src/assets/js/usage.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * Usage class.
+ *
+ * @summary This file handles usage statistics.
+ *
+ * @since SINCEVERSION
+ */
+
+/* global jQuery,ga,gtag */
+
+var BOLDGRID = BOLDGRID || {};
+
+BOLDGRID.LIBRARY = BOLDGRID.LIBRARY || {};
+
+(function($) {
+	'use strict';
+
+	var self;
+
+	/**
+	 * Total Upkeep Usage.
+	 *
+	 * @since SINCEVERSION
+	 */
+	BOLDGRID.LIBRARY.Usage = {
+		/**
+		 * Add our client id to all boldgrid.com links.
+		 *
+		 * @since SINCEVERSION
+		 */
+		addClientId: function() {},
+
+		/**
+		 * Prepend all boldgrid.com links with our user's GA client id.
+		 *
+		 * @since SINCEVERSION
+		 *
+		 * @link https://www.analyticsmania.com/post/google-analytics-cross-domain-tracking-with-google-tag-manager/
+		 * @link https://developers.google.com/analytics/devguides/collection/gtagjs/cross-domain
+		 */
+		prependClientId: function() {
+			var clientId = self.getClientId(),
+				part;
+
+			// Abort if we can't get a client id.
+			if ('' === clientId) {
+				return;
+			}
+
+			part = '_ga=' + clientId;
+
+			$('a[href*="www.boldgrid.com"]').each(function() {
+				var $anchor = $(this),
+					url = $anchor.attr('href');
+
+				/*
+				 * Add our client id to the url.
+				 *
+				 * @link https://stackoverflow.com/questions/486896/adding-a-parameter-to-the-url-with-javascript
+				 */
+				url =
+					url.indexOf('?') != -1
+						? url.split('?')[0] + '?' + part + '&' + url.split('?')[1]
+						: url.indexOf('#') != -1
+						? url.split('#')[0] + '?' + part + '#' + url.split('#')[1]
+						: url + '?' + part;
+
+				$anchor.attr('href', url);
+			});
+		},
+
+		/**
+		 * Get the user's client id.
+		 *
+		 * @since SINCEVERSION
+		 */
+		getClientId: function() {
+			var clientId = '';
+
+			ga.getAll().forEach(tracker => {
```

Comment:
`usage.js?ver=20191213:80 Uncaught TypeError: ga.getAll is not a function
    at Object.getClientId (usage.js?ver=20191213:80)
    at Object.prependClientId (usage.js?ver=20191213:42)
    at HTMLDocument.<anonymous> (usage.js?ver=20191213:166)`

Reply:
This has been fixed in another commit - https://github.com/BoldGrid/library/commit/1fd4601d27cfec75b67fe2a98e1e6d69449d6b4d